vnext/include/ReactUWP/Views/ViewManagerBase.h

```diff
@@ -62,6 +62,7 @@ class REACTWINDOWS_EXPORT ViewManagerBase : public facebook::react::IViewManager
 
   virtual void UpdateProperties(ShadowNodeBase *nodeToUpdate, const folly::dynamic &reactDiffMap);
 
+  virtual void DispatchCommand(XamlView viewToUpdate, const std::string &commandId, const folly::dynamic &commandArgs);
   virtual void DispatchCommand(XamlView viewToUpdate, int64_t commandId, const folly::dynamic &commandArgs);
```

Comment:
>XamlView viewToUpdate [](start = 31, length = 21)

XamlView is a smart pointer with additional info.
The best approach is to pass it by const& like we do for the std::string.

Reply:
I agree. I'll try to bulk change everything in a followup PR instead of doing it just for this.